src/Integration.Vsix/Commands/HelpMenu/AboutCommand.cs

```diff
@@ -26,8 +26,6 @@ internal class AboutCommand : VsCommandBase
     {
```

Comment:
Not sure how to test this class best now, mostly because I don't really like how I set it up with the AboutDialog clsas, let me know what you think

Reply:
Internal testing ctor that receives a "Create" delegate and returns a mock. But it's ok, it doesn't do much, we can skip the test IMO.